hexrd/instrument/detector.py

```diff
@@ -126,12 +125,10 @@ def angles_to_cart(self, tth_eta,
             The (n, 2) array on the n input coordinates in the .
 
         """
-        pass
```

Comment:
This pass is necessary, please either do not remove it, or raise a `NotImplementedError`.